src/EditorFeatures/CSharpTest/Completion/CompletionProviders/DeclarationNameCompletionProviderTests.cs

```diff
@@ -1181,6 +1181,50 @@ class Test
             await VerifyItemExistsAsync(markup, "tokens");
         }
 
+        [WorkItem(37366, "https://github.com/dotnet/roslyn/issues/37366")]
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async Task PluralizeSpan()
+        {
+            var markup = @"
+using System;
+
+class Test
+{
+    void M(Span<Test> $$) { }
```

Comment:
This test doesn't seem to be run against a runtime that have a `Span<T>` definition. What's the correct way to fix that? (besides having to clone part of the implementation from dotnet/runtime repo)